project_national/fact_sheets_category_4.yml

```diff
@@ -0,0 +1,276 @@
+              - State|KS
+              - State|MN
+              - State|IA
+              - State|MO
+              - State|WI
+              - State|IL
+              - State|MI
+              - State|IN
+              - State|OH
+              - State|WV
+              - State|PA
+              - State|NJ
+              - State|NY
+              - State|CT
+              - State|RI
+              - State|MA
+              - State|VT
+              - State|NH
+              - State|ME
+
+  - upgrade_name: ENERGYSTAR Room AC
+    options:
+      - &energystar_room_ac
+        option: HVAC Cooling Efficiency |Room AC, EER 12.0
+        apply_logic:
+          - or:
+            - HVAC Cooling Efficiency|Room AC, EER 8.5
+            - HVAC Cooling Efficiency|Room AC, EER 9.8
+            - HVAC Cooling Efficiency|Room AC, EER 10.7
+
+  - upgrade_name: ENERGYSTAR Central AC
+    options:
+      - &energystar_central_ac
+        option: HVAC Cooling Efficiency |AC, SEER 15.2
+        apply_logic:
+          - or:
+            - HVAC Cooling Efficiency|AC, SEER 8
+            - HVAC Cooling Efficiency|AC, SEER 10
+            - HVAC Cooling Efficiency|AC, SEER 13
+            - HVAC Cooling Efficiency|AC, SEER 14
+            - HVAC Cooling Efficiency|AC, SEER 15
+
+  - upgrade_name: HPWH
+    options:
+      - &water_heater_fuel_type
+        option: Water Heater Fuel|Electricity
+      - &water_heater_efficiency_electric_heat_pump_50_gal_3_pt_45_uef
+        option: Water Heater Efficiency|Electric Heat Pump, 50 gal, 3.45 UEF
+        apply_logic:
+          - or:
+            - Occupants|0
+            - Occupants|1
+            - Occupants|2
+        costs:
+          - value: 100.0 #TBD
+            multiplier: Size, Water Heater (gal)
+        lifetime: 12
+      - &water_heater_efficiency_electric_heat_pump_66_gal_3_pt_35_uef
+        option: Water Heater Efficiency|Electric Heat Pump, 66 gal, 3.35 UEF
+        apply_logic:
+          - or:
+            - Occupants|3
+        costs:
+          - value: 100.0 #TBD
+            multiplier: Size, Water Heater (gal)
+        lifetime: 12
+      - &water_heater_efficiency_electric_heat_pump_80_gal_3_pt_45_uef
+        option: Water Heater Efficiency|Electric Heat Pump, 80 gal, 3.45 UEF
+        apply_logic:
+          - or:
+            - Occupants|4
+            - Occupants|5
+            - Occupants|6
+            - Occupants|7
+            - Occupants|8
+            - Occupants|9
+            - Occupants|10+
+        costs:
+          - value: 100.0 #TBD
+            multiplier: Size, Water Heater (gal)
+        lifetime: 12
+
+  - upgrade_name: 96% AFUE Natural Gas Furnace
+    options:
+      - &Furnace_fuel_type
+        option: HVAC Heating Type And Fuel|Natural Gas Fuel Furnace
+        apply_logic:
+          - and: 
+            - or:
+              - Heating Fuel|Natural Gas
+              - Water Heater Fuel|Natural Gas # Do we need more to represent any natural gas usage?
```

Comment:
I think so, if we intend to only replace if there's existing gas infrastructure. There's cooking, dryer, misc hot tub spa, misc pool heater, misc gas fireplace, misc gas grill, misc gas lighting

Reply:
Done